### PR TITLE
Add Tokopedia Presence

### PR DESCRIPTION
**Pull request by WilloIzCitron**
Proof of working:
Screenshot (156)
Screenshot (158)

### CODE REVIEW

**Comment by WilloIzCitron** [comment]
#2736

File: websites/T/Tokopedia/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "author": {
+        "name": "WilloIzCitron",
+        "id": "735410445409845251"
+    },
+    "contributors": [
+        {
+            "name": "WilloIzCitron",
+            "id": "735410445409845251"
+        }
+    ],
```
Comment (Bas950) [inline websites/T/Tokopedia/dist/metadata.json:12]: dont add yourself as contributor

File: websites/T/Tokopedia/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "service": "Tokopedia",
+    "description": {
+        "en": "Rich presence for Tokopedia - The Indonesian E-Commerce Site"
```
Comment (Bas950) [inline websites/T/Tokopedia/dist/metadata.json:15]: This should be the description of Tokopedia not the description of the presence

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/T/Tokopedia/dist/metadata.json (hunk 1 above)
Comment (WilloIzCitron) [reply websites/T/Tokopedia/dist/metadata.json:12]: ok
imma fix it

File: websites/T/Tokopedia/dist/metadata.json (hunk 2 above)
Comment (WilloIzCitron) [reply websites/T/Tokopedia/dist/metadata.json]: and it too

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**